templates/CRM/Financial/Form/Search.tpl

```diff
@@ -242,6 +242,8 @@ CRM.$(function($) {
         if (response.status == 'record-updated-success') {
           CRM.alert(listRecords(records), op == 'delete' ? {/literal}'{ts escape="js"}Deleted{/ts}' : '{ts escape="js"}Updated{/ts}'{literal}, 'success');
           batchSelector.fnDraw();
+	  //Redirect CRM-18169
+          window.location.href = CRM.url('civicrm/financial/financialbatches', 'reset=1&batchStatus=' + response.status_id);
```

Comment:
This doesn't make sense to me. Why would you refresh the datatable *and* redirect to a different page?